src/runtime/data/kata-collect-data.sh

```diff
@@ -18,8 +24,8 @@ typeset -r containerd_shim_v2=$(command -v "$containerd_shim_v2_name" 2>/dev/nul
 typeset -r kata_monitor_name="kata-monitor"
 typeset -r kata_monitor=$(command -v "$kata_monitor_name" 2>/dev/null)
 
-typeset -r issue_url="@PROJECT_BUG_URL@"
-typeset -r script_version="@VERSION@ (commit @COMMIT@)"
+typeset -r issue_url="$PROJECT_BUG_URL"
```

Comment:
Any reason for not `typeset -r issue_url=https://github.com/kata-containers/kata-containers/issues/new` here?

Reply:
No